pyproject.toml

```diff
@@ -99,14 +99,17 @@ select = [
 ]
```

Comment:
See https://docs.astral.sh/ruff/configuration/.

`select` should be in the `[tool.ruff.lint]` section and `[tool.ruff.per-file-ignores]` should be `[tool.ruff.lint.per-file-ignores]`.

I think it also make sense to sort the section by the following order:
```
[tool.ruff]

[tool.ruff.lint]

[tool.ruff.lint.per-file-ignores]

[tool.ruff.format]

# tool-specific configurations
[tool.ruff.isort]

[tool.ruff.pycodestyle]
```

We probably also want to require ruff>=0.1.3.

Reply:
Hmm, did the change the TOML config settings recently? I see pycodestyle is now under [`tool.ruff.lint.pycodestyle`](https://docs.astral.sh/ruff/settings/#pycodestyle) too. Let me fix these up.

Reply:
Done in 8c58089627dd64933ed304e9b1ff6c71729c43d1.